adam-cli/src/test/resources/contigs.fa

```diff
@@ -0,0 +1,836 @@
+>chromosome:GRCh38:6:29941260:29966260:1
```

Comment:
I believe the default is 10k, so at 25k each both of the test contigs should break into 2 1/2 fragments.